python-api/gstswitch/helpers.py

```diff
@@ -87,7 +90,7 @@ def audio_port(self, audio_port):
     @property
     def running_tests_video(self):
         """Get the currently running test video list"""
-        return self._running_tests_video
+        return list(self._running_tests_video)
```

Comment:
Quoting http://stackoverflow.com/a/626871/1659732:
> Apart from tuples being immutable there is also a semantic distinction that should guide their usage. Tuples are heterogeneous data structures (i.e., their entries have different meanings), while lists are homogeneous sequences. **Tuples have structure, lists have order.**

The returned list is a copy of the original one and I can see reasons for it to be mutable, for example if you want to pop elements of it in the application.